Create supplementary files

Merk1998 opened the pull request:
src/main/resources/defaults-storage-driver-kafka.yml
    src/main/resources/META-INF/services/com.emc.mongoose.base.Extension
    src/main/resources/config-schema-storage-driver-kafka.yml

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Darya Merkureva mdp1998@icloud.com

Merk1998 committed:
Create supplementary files
    src/main/resources/defaults-storage-driver-kafka.yml
    src/main/resources/META-INF/services/com.emc.mongoose.base.Extension
    src/main/resources/config-schema-storage-driver-kafka.yml

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Darya Merkureva mdp1998@icloud.com

akurilov commented on src/main/resources/META-INF/services/com.emc.mongoose.base.Extension:
```diff
@@ -0,0 +1 @@
+com.emc.mongoose.storage.driver.kafka.KafkaStorageDriverExtension
```
diff suggests to add a new line

akurilov replied:
the name of the file should be
com.emc.mongoose.base.env.Extension
instead

akurilov commented on src/main/resources/defaults-storage-driver-kafka.yml:
```diff
@@ -0,0 +1,24 @@
+    linger: 0
+    node:
+      addrs:
+      port: 9092
```
missing new line

Merk1998 committed:
Fix some issues

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Darya Merkureva mdp1998@icloud.com

Merk1998 committed:
Update defaults-storage-driver-kafka.yml
